style-dictionary/visual-refresh/shadows.ts

```diff
@@ -5,30 +5,21 @@ import { StyleDictionary } from '../utils/interfaces';
 import { expandColorDictionary } from '../utils';
 
 const tokens: StyleDictionary.ShadowsDictionary = {
-  shadowContainer: {
-    light: '0px 0px 1px 1px #e9ebed, 0px 1px 8px 2px rgba(0, 7, 22, 0.12)', // 1px grey-200 faux border
-    dark: '0px 1px 8px 2px rgba(0, 7, 22, 0.6)',
-  },
-  shadowContainerStacked: {
-    light:
-      '-1px 1px 1px 0px #e9ebed, 1px 1px 1px 0px #e9ebed, 0px 9px 8px -7px rgb(0 7 22 / 12%), 8px 0px 8px -7px rgb(0 7 22 / 12%), -8px 0px 8px -7px rgb(0 7 22 / 12%)',
-    dark: '0px 9px 8px -7px rgb(0 7 22 / 60%), 8px 0px 8px -7px rgb(0 7 22 / 60%), -8px 0px 8px -7px rgb(0 7 22 / 60%)',
-  },
   shadowContainerActive: {
     light: '0px 1px 1px 1px #e9ebed, 0px 6px 36px #0007161a',
     dark: '0px 1px 1px 1px #192534, 0px 6px 36px #000716',
```

Comment:
We want to update the shadow for board component containers in their dragged state, right? At least for dark mode?